tests/unit/ssg-module/test_controls.py

```diff
@@ -113,6 +139,8 @@ def test_profile_resolution_extends():
     assert "var_password_pam_ocredit" in high_profile.variables
     assert high_profile.variables["var_password_pam_ocredit"] == "2"
 
+
+@pytest.mark.skip
```

Comment:
The `abcd-all.profile` file is not there.

Reply:
Why was it removed?

Reply:
Aha, I see, It accidentally wasn't committed in #5, I have added a new commit to #5. I'm sorry for confusion. You can re-base and remove it.